x-pack/plugins/rule_registry/server/alert_data_client/alerts_client_factory.test.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Request } from '@hapi/hapi';
+
+import { AlertsClientFactory, AlertsClientFactoryProps } from './alerts_client_factory';
+import { ElasticsearchClient, KibanaRequest } from 'src/core/server';
+import { loggingSystemMock } from 'src/core/server/mocks';
+import { securityMock } from '../../../security/server/mocks';
```

Comment:
Doesn't this create a dependency on the security plugin? If so it needs to be specified in kibana.json and tsconfig.json.